wasp/apps/word_clock.py

```diff
@@ -0,0 +1,193 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2020 Daniel Thompson
+# Copyright (C) 2021 Brendan Sleight
+
+
+"""Digital clock
```

Comment:
This looks out of date. It needs to be different from the other existing clocks.